source/fetch_pbd.rst

```diff
@@ -0,0 +1,131 @@
+Tutorial: Fetch Programming by Demonstration
+============================================
+
+This repository is based on PR2 Programming by Demonstration. This version is for Fetch. It will not work on Freight because Freight does not have an arm.
```

Comment:
Since this is going on docs.fetchrobotics, you should add a link here to the repo being referred to